x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/editor.tsx

```diff
@@ -306,6 +317,13 @@ export const CustomUrlEditor: FC<CustomUrlEditorProps> = ({
               />
             </EuiFormRow>
           )}
+        {(type === URL_TYPE.KIBANA_DASHBOARD || type === URL_TYPE.KIBANA_DISCOVER) &&
```

Comment:
For Discover custom URLs, can you tell if the target data view has a timestamp field? If so, you should hide the time range selector if the target data view does not have a timestamp field.

Reply:
Due to the bug where the source index timefield doesn't get mapped automatically in the destination index (https://github.com/elastic/kibana/issues/155432) - I think this should go in after the fix. Otherwise this won't show up most of the time for discover links. 🤔 What do you think?